backend/migrations/versions/c6d01930179d_add_backfill_read_folder_permissions.py

```diff
@@ -30,6 +36,53 @@ def get_session():
     return session
 
 
+Base = declarative_base()
+
+
+class Dataset(Resource, Base):
```

Comment:
Do we need all of the below columns / do they even exist on base dataset table anymore?

Can we just query `dataset` table for
`SamlAdminGroupName`
`stewards`
`datasetUri`

Reply:
Same as above; I think it makes it clear that importing the current DatasetBase or S3Dataset is not the same as using Dataset at that moment in time